src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -197,7 +231,7 @@ export default class GitOpsConfiguration extends Component<GitOpsProps, GitOpsSt
     }
 
     render() {
-        let key: "gitHubOrgId" | "gitLabGroupId" = this.state.tab === GitProvider.Github ? 'gitHubOrgId' : 'gitLabGroupId';
+        let key: "gitHubOrgId" | "gitLabGroupId" | "azureOrgId" = this.state.tab === GitProvider.Github ? 'gitHubOrgId' : this.state.tab === GitProvider.Azure_devops ? 'azureOrgId' : 'gitLabGroupId';
```

Comment:
extract out type of key
extract  it in if else
gitLabGroupId is default.. is that correct?

Reply:
1. fixed type
2.  abstract out the logic in a separate function
3. fixed: We are keeping `'gitHubOrgId'` as the default